test/testcase.jl

```diff
@@ -66,7 +67,18 @@ function testcase(
         for file in filesref
             dataref = CSV.File(joinpath(refpostprodir, file); header=1) |> DataFrame
             data    = CSV.File(joinpath(postprodir, file); header=1) |> DataFrame
-            data    = data[1:size(dataref, 1), :]
+            if (size(data, 1) < size(dataref, 1))
```

Comment:
Let's replace this by another argument to `testcase`: A list of files for which to exclude the check for equality in the number of rows. If the file matches the expression given, we can skip the `@test` on line 77 and just test on the first `min(size(data, 1), size(dataref, 1)` rows of `data` and `data_ref`. This way we aren't doing this slightly shady data copying to match rows. This parameter will default to being empty (check for all files) but should include usually the `estimator.csv` output as well as all files (support wildcard?) for eigenmode simulations. What do you think?